moztelemetry/dataset.py

```diff
@@ -41,6 +42,32 @@ def _group_by_size_greedy(obj_list, tot_groups):
     return groups
 
 
+def _group_by_equal_size(obj_list, tot_groups, threshold=pow(2, 32)):
```

Comment:
Add a docstring like `_group_by_size_greedy`.

Reply:
Could you clarify what you mean by this? Is there a particular format I should follow that is generally used, or should I mostly copy the style of the _group_by_size_greedy format as closely as possible?

Reply:
Docstrings are the pythonic way of docmenting code. See [PEP 257](https://www.python.org/dev/peps/pep-0257/) for some background. Concretely, we have [autogenerated doc pages](https://python-moztelemetry.readthedocs.io/en/latest/) for this repository that are created from docstrings.

We use [sphinx](http://www.sphinx-doc.org/en/master/) to package the documentation, you can see the `/docs` folder for how the build process works. If you follow the general style of the comments that exist in the repo, it should probably work fine. Here's [a website](https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html#auto-documented-class-method-1) that has some examples on some of the syntax like `:param <argument name>:`. I've also provided an example with the elements of a good docstring. 

```python
def function(arg1, arg2):
    """Here's a short description.

    Here's a more detailed description. It can have multiple sentences. It
    may describe context and why you may use this function.

    :param arg1: the type and description
    :param arg2: the type and description
    :return: a type and description
    """
    pass
```